draftlogs/7005_add.md

```diff
@@ -0,0 +1 @@
+ - Add property x_shift/y_shift for shapes/selections referencing (multi-)category axes [[#7005](https://github.com/plotly/plotly.js/pull/7005)]
```

Comment:
Let's update this description using the final attribute names.

Reply:
done